numpy/random/mtrand/mtrand.pyx

```diff
@@ -1356,11 +1682,13 @@ cdef class RandomState:
         """
```

Comment:
Just curious: in light of the recent discussion surrounding half-open and closed intervals, is it worth deprecating this function in this release?